src/CMakeLists.txt

```diff
@@ -615,6 +615,11 @@ if(BUILD_CLIENT)
 	if (USE_SPATIAL)
 		target_link_libraries(${PROJECT_NAME} ${SPATIAL_LIBRARY})
 	endif()
+	if(BUILD_UNITTESTS)
+		target_compile_definitions(${PROJECT_NAME} PRIVATE
+			UNITTEST_ASSETS_DIRECTORY=${UNITTEST_ASSETS_DIRECTORY}
```

Comment:
now I do have comments:
baking build time paths into into the binary isn't good, see 133f706bf3e4c001b8f42179af3f070cc1940d1e and #12801

Reply:
moved to devtest in https://github.com/minetest/minetest/pull/14557/commits/50338874f193305fcf43262e14fa15a78721e02d

catch2 test skipping requires catch2 v3 hence depends on https://github.com/minetest/minetest/pull/14735